src/summary/history/programs/programs.resource.tsx

```diff
@@ -0,0 +1,18 @@
+import { openmrsObservableFetch } from "@openmrs/esm-api";
+import { Observable } from "rxjs";
+import { map, take } from "rxjs/operators";
+
+
+export function performPatientProgramsSearch(
+  patientID: string
+): Observable<PatientPrograms[]> {
+  return openmrsObservableFetch(
+    `/ws/rest/v1/programenrollment?patient=${patientID}`
```

Comment:
There is no fhir api for getting program related data. I have created this ticket,
https://issues.openmrs.org/projects/MF/issues/MF-121 to remind us to make the change.